opensrp-chw/build.gradle

```diff
@@ -51,6 +51,12 @@ android {
         buildConfigField "int", "MAX_SYNC_RETRIES", '3'
         buildConfigField "long", "BUILD_TIMESTAMP", System.currentTimeMillis() + "L"
         buildConfigField "boolean", "IS_SYNC_SETTINGS", "false"
+        buildConfigField "String", "THINKMD_BASE_URL", '"https://thinkmd-v4-app-b-ona.herokuapp.com/"'
+        buildConfigField "String", "THINKMD_END_POINT", '"https://fhir-v4-staging-thinkmd-tech.herokuapp.com/fhir"'
+        buildConfigField "String", "PATIENT_IDENTIFIER", '"https://fhir.smartregister.org/thinkMD_id"'
+        buildConfigField "String", "ENCOUNTER_IDENTIFIER", '"https://fhir.smartregister.org/encounterUuid"'
+        buildConfigField "String", "MESSAGE_EVENT_URL", '"http://hl7.org/fhir/ValueSet/message-events"'
+        buildConfigField "String", "PRACTITIONER_COMMUNICATION_CODING_SYSTEM", '"urn:ietf:bcp:47"'
```

Comment:
I think will not be changing per thinkmd integration and should be defined as constants on `ThinkMDConfig`

Reply:
On ThinkMDConfig we have private variables with getters and setters.

Reply:
Yes lets only have 
```
 buildConfigField "String", "THINKMD_BASE_URL", '"https://thinkmd-v4-app-b-ona.herokuapp.com/"'
 buildConfigField "String", "THINKMD_END_POINT", '"https://fhir-v4-staging-thinkmd-tech.herokuapp.com/fhir"'
```
As the only variables defined via build config